.github/workflows/release-plz.yml

```diff
@@ -0,0 +1,27 @@
+name: Release-plz
+
+permissions:
+  pull-requests: write
+  contents: write
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  release-plz:
+    name: Release-plz
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Install Rust toolchain
+        uses: dtolnay/rust-toolchain@stable
```

Comment:
for consistency could we use the same style as the other workflows?

https://github.com/cargo-bins/cargo-binstall/blob/main/.github/workflows/release.yml#L33

Reply:
```suggestion
        run: rustup toolchain install stable --no-self-update --profile minimal
```